arquillian/adapter/src/main/java/org/wildfly/swarm/arquillian/adapter/UberjarSimpleContainer.java

```diff
@@ -201,10 +204,16 @@ public void start(Archive<?> archive) throws Exception {
         // check for "org.wildfly.swarm.allDependencies" flag
         // see DependenciesContainer#addAllDependencies()
         if (archive instanceof DependenciesContainer) {
-            DependenciesContainer depContainer = (DependenciesContainer) archive;
-            if (depContainer.hasMarker("org.wildfly.swarm.allDependencies")) {
+            DependenciesContainer<?> depContainer =  (DependenciesContainer<?>) archive;
+            if (depContainer.hasMarker(DependenciesContainer.ALL_DEPENDENCIES_MARKER)) {
                 munge(depContainer, declaredDependencies);
             }
+        } else if (archive instanceof WebArchive) {
```

Comment:
Yep, for instance:
https://github.com/wildfly-swarm/wildfly-swarm/blob/master/arquillian/adapter/src/main/java/org/wildfly/swarm/arquillian/adapter/DefaultDeploymentScenarioGenerator.java#L61